doc/source/ray-core/handling-dependencies.rst

```diff
@@ -402,6 +402,40 @@ Example:
   {"pip": ["torch", "ray[serve]"],
   "env_vars": {"A": "a", "B": "new", "C", "c"}}
 
+.. _runtime-env-faq:
+
+Frequently Asked Questions
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Are environments installed on every node?
+"""""""""""""""""""""""""""""""""""""""""
+
+If specifying your runtime environment for the job, in ``ray.init(runtime_env=...)``, then the environment will be installed on every node.  See :ref:`Per-Job <rte-per-job>` for more details.
```

Comment:
When we say "job" here, do we mean a job-submission job? What happens if I'm not using job submission? It might be clearer to directly say "If a runtime environment is specified in ``ray.init(runtime_env=...)``, then the environment will..."

Reply:
Yeah it's a little unfortunate that we have the "old" job terminology and the new "ray job submit" jobs being discussed in the same doc.    I think your edit will make it clearer